zetaclient/chains/evm/observer/inbound.go

```diff
@@ -344,21 +352,24 @@ func (ob *Observer) observeERC20Deposited(
 
 	// collect and sort events by block number, then tx index, then log index (ascending)
 	events := make([]*erc20custody.ERC20CustodyDeposited, 0)
-	for _, log := range logs {
+	for _, ethlog := range ethlogs {
 		// sanity check tx event
-		err := common.ValidateEvmTxLog(&log, addrCustody, "", common.TopicsDeposited)
+		err := common.ValidateEvmTxLog(&ethlog, addrCustody, "", common.TopicsDeposited)
 		if err != nil {
 			continue
 		}
-		event, err := erc20custodyContract.ParseDeposited(log)
+		event, err := erc20custodyContract.ParseDeposited(ethlog)
 		if err == nil {
 			events = append(events, event)
 			continue
 		}
 		ob.Logger().Inbound.Warn().
 			Err(err).
-			Msgf("ObserveERC20Deposited: invalid Deposited event in tx %s on chain %d at height %d",
-				log.TxHash.Hex(), ob.Chain().ChainId, log.BlockNumber)
+			Str(logs.FieldMethod, "observeERC20Deposited").
+			Str("tx", ethlog.TxHash.Hex()).
```

Comment:
i think we have FieldNonce FieldTx etc for these common ones

Reply:
The `Int64("chain", ob.Chain().ChainId)` is redundant.